### PR TITLE
Init R package

### PR DESCRIPTION
**Pull request by arkadiuszbeer**

### CODE REVIEW

File: tests/testthat.R [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pkg_name <- "teal"
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply tests/testthat.R:1]: Sorry and thanks for highlighting it

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* @insightsengineering/nest-sme
+.github/workflows/* @insightsengineering/nest-automation
+.gitlab-ci.yml @insightsengineering/nest-automation
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply .github/CODEOWNERS]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+**OS / Environment**
+
+<!--
+Provide all relevant information below, e.g. target OS versions, docker image, etc.
+-->
+
+- OS: [e.g. Windows 10, Ubuntu 20.04, Centos 8]
+- Docker Image [e.g. rocker/verse:4.1.0]
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+As a [persona], I [want to], [so that].
+
+**Additional Information**
+
+<!--
+Describe how the feature would be used, why it is needed and what it would solve.
+
+**HINT:** You can paste https://gist.github.com links for larger files.
+-->
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply .github/ISSUE_TEMPLATE/feature_request.md]: with yaml files. Done

File: _pkgdown.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+url: https://insightsengineering.github.io/nestcolor
+
+template:
+  params:
+    bootswatch: cerulean
+    ganalytics: UA-125641273-1
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply _pkgdown.yml]: done

File: inst/WORDLIST [hunk 6]
```diff
@@ -0,0 +1,6 @@
+### Added from global WORDLIST
+biomarker
+NEST
+reproducibility
+Roche
+RSConnect
```
[the remark this answers is not in the document]
Comment (arkadiuszbeer) [reply inst/WORDLIST]: Taken fron r.pkg.template

**Review by dinakar29 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**